tests/go-stdlib-json.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+    "fmt"
+    "encoding/json"
+    "os"
+    "io/ioutil"
+    "time"
+)
+
+
+func main() {
+    if len(os.Args) != 3 {
+        fmt.Println("Command format: go-stdlib-json [small-json] [large-json]")
+    }
+
+    small := os.Args[1]
+    big := os.Args[2]
+
+    // Test loading
+    big_duration, total_big_duration := load_json(big, 96)
+    small_duration, total_small_duration := load_json(small, 648494)
+    fmt.Printf("go-stdlib-json-deserialize,%v,%v,%v,%v\n", small_duration,
+               total_small_duration, big_duration, total_big_duration)
+
+    // Test serializing
+    big_duration, total_big_duration = dump_json(big, 67)
+    small_duration, total_small_duration = dump_json(small, 786217)
+    fmt.Printf("go-stdlib-json-serialize,%v,%v,%v,%v\n", small_duration,
+               total_small_duration, big_duration, total_big_duration)
+}
+
+
+func load_json(jFile string, iterations int) (float64, float64) {
+    jsonBytes, err := ioutil.ReadFile(jFile)
+    if err != nil {
+        fmt.Println("Error! Could not open file ", jFile)
+    }
+
+    start := time.Now()
+    // Output data
+    var data map[string]interface{}
```

Comment:
This doesn't get timed in other cases. Probably not a noticeable amount of time, but symmetry...